### PR TITLE
Add a brand and colour for archive

### PR DESCRIPTION
**Pull request by jimjohnsonrollings**
Part of #2427 and #2425 

**Overall change:**
Add brand and colour for Archive

**Code changes:**

* Add colour C_ARCHIVE_BLUE
* Update tests for colours
* Bump psammead-styles version number
* Add SVG for Archive
* Update tests for SVGs
* Bump psammead-assets version number
* Add font-families for Archive to use Reith Latin
* Update tests for fonts

**Testing notes**
Both colour and SVG can be seen in Storybook

Screen Shot 2019-11-25 at 16 59 52
Screen Shot 2019-11-25 at 17 00 24

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Automated jest tests added (for new features) or updated (for existing features)
- [x] This PR requires manual testing

### CODE REVIEW

**Review by rhenshaw56 — APPROVED** [review]

**Review by RayNjeri — APPROVED** [review]

**Review by EinsteinNjoroge — APPROVED** [review]
👍

**Review by jroebu14 — APPROVED** [review]
Looks good

File: packages/utilities/psammead-assets/src/svgs/archive.jsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+const archive = {
+  group: (
+    <g
+      fillRule="evenodd"
+      stroke="#fff"
```
Comment (sareh) [inline packages/utilities/psammead-assets/src/svgs/archive.jsx]: Changing this to black ensures that without CSS, the logo renders correctly, with a black stroke. This matches implementation in our other Brand SVGs. 

```suggestion
      stroke="#000"
```

After the change, looking at http://localhost:8180/iframe.html?id=utilities-svgs-brand-svgs--archive with CSS disabled, gives us this logo: 
Screenshot 2019-11-26 at 10 17 25

File: packages/utilities/psammead-assets/src/__snapshots__/svgs.test.jsx.snap [hunk 2]
```diff
@@ -117,6 +117,33 @@ exports[`arabic SVG should render correctly 1`] = `
 </svg>
 `;
 
+exports[`archive SVG should render correctly 1`] = `
+<svg
+  viewBox="207 24"
+>
+  <g
+    fill-rule="evenodd"
+    stroke="#fff"
```
Comment (sareh) [inline packages/utilities/psammead-assets/src/__snapshots__/svgs.test.jsx.snap]:
```suggestion
    stroke="#000"
```

File: packages/utilities/psammead-assets/src/__snapshots__/svgs.test.jsx.snap [hunk 3]
```diff
@@ -117,6 +117,33 @@ exports[`arabic SVG should render correctly 1`] = `
...
+    stroke-width=".335"
+    style="stroke: #fff;"
```
Comment (sareh) [inline packages/utilities/psammead-assets/src/__snapshots__/svgs.test.jsx.snap:128]: Looking at the accessibility of the SVG - when viewing with Firefox's colours being changed, the styling looks good - the outline of the SVG is correctly visible and fill/background colours also good too.
Screenshot of SVG in Firefox

**Review by sareh — COMMENTED** [review]
Looks great. Just one small comment.

**Review by sareh — APPROVED** [review]
👍 Thanks for the update.

**Review by HarveyPeachey — APPROVED** [review]
LGTM

**Comment by jamesbrumpton** [comment]
lgtm

**Pull request merged**